src/main/clojure/clara/tools/testing_utils.cljc

```diff
@@ -0,0 +1,89 @@
+#?(:clj
+   (ns clara.tools.testing-utils
+     "Internal utilities for testing clara-rules and derivative projects.  These should be considered experimental
+  right now from the perspective of consumers of clara-rules, although it is possible that this namespace 
+  will be made part of the public API once its functionality has proven robust and reliable.  The focus, however,
+  is functionality needed to test the rules engine itself."
+     (:require [clara.rules.update-cache.core :as uc]
+               [clara.rules.update-cache.cancelling :as ca]
+               [clara.rules.compiler :as com]
+               [clara.macros :as m]
+               [clara.rules.dsl :as dsl]))
+   :cljs
+   (ns clara.tools.testing-utils
+     (:require [clara.rules.update-cache.core :as uc])))
+
+#?(:clj
```

Comment:
Similar to our past discussion on macros in cljs, this shouldn't be specified in a read-conditional. It isn't platform specific. It is used in cljs testing.